.github/workflows/app-workflow.yml

```diff
@@ -192,16 +192,19 @@ jobs:
       - name: Run tests
         run: npm run e2e:tests
       - name: Upload artifacts (screenshots)
+        if: ${{ always() }}
```

Comment:
what does this does? 😅

Reply:
This uploads the artifacts even if the test fails 👍